### DISCUSSION TITLE
Is there a way to close multiple tabs at the same time?

### DISCUSSION THREAD

**Discussion by alkim0** [qutebrowser/qutebrowser#7328] [Q&A] [answered]
I would expect "3d" to close the next three tabs, but it closes the third tab instead. How do I get it to close three tabs?

**Comment by The-Compiler** [comment, marked as answer]
There's the `:repeat` command, so you can do `:repeat 3 tab-close`. You can also `:bind d repeat 1 tab-close` since the count given to it gets multiplied with the argument.

**Reply by jbpoittevin** [reply]
Thank you for suggesting :repeat, I was searching how to do it with `set-cmd-text` without success.

The idea of repeating `tab-close` command with a count prefix came with trying to treat the list of tab in qutebrowser like a line in vim.

Here is what I obtained:

```python
config.unbind('d')
config.bind('dd', 'close')
config.bind('dw', 'repeat 1 tab-close')
config.bind('dh', 'repeat 1 tab-prev ;; tab-close --next')
config.bind('dl', 'repeat 1 tab-next ;; tab-close --prev')
config.bind('d$', 'tab-only --prev')
config.bind('d^', 'tab-only --next')
```
